package.json

```diff
@@ -9,7 +9,10 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint src",
+    "lint": "eslint src playground",
+    "playground:prepare": "npm run prepublishOnly && babel playground --out-dir .playground-lib",
+    "playground:scrape": "node .playground-lib/scrape.js",
+    "playground:setup": "node .playground-lib/setup.js",
```

Comment:
Adding `babel-cli` make sense, but I didn't want to do major changes to the current project structure. If you want I can change the workflow to use babel-cli. let me know.

Reply:
@esakal I don't think this is a major change, we don't even have to add another dependency to use it (it's included under `babel` module).

> A prerequisite will be to make sure that we can add breakpoints somewhere when using babel-cli, otherwise we are missing the point of having playground support

Good point, but I'm sure this is doable.